README.md

```diff
@@ -10,11 +10,11 @@
 * Documentation: https://docs.hortonworks.com/HDPDocuments/Cloudbreak/Cloudbreak-2.7.0/index.html
 
 # Local Development Setup
-As of now this document is focusing on setting up your development environment on OSX. You'll need brew to install certain components in case you don't have them already. To get brew please follow the install instructions on the brew homepage: https://brew.sh
+As of now this document focuses on setting up your development environment on macOS. You'll need Homebrew to install certain components in case you don't have them already. To get Homebrew please follow the install instructions on the Homebrew homepage: https://brew.sh
 
-As a prerequisite you need to have Java 11 installed. You can get it from [here](https://jdk.java.net/11/).
+As a prerequisite you need to have Java 11 installed. You can get it from [here](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html).
```

Comment:
Given the license restrictions of Oracle Java 11+, is it okay to send people there as opposed to something like OpenJDK? I mean, Oracle used to be the standard choice of JDK (except for some bizarre cases where the customer insisted on another vendor like IBM proprietary stuffs), but this might be changing with the new licensing conditions.

Reply:
Good question. I know that for _development_ purposes, you are still free to use the Oracle JDK, so for this section "Local Development Setup" it's OK. But, I agree that the waters are muddied now. I'll rephrase and find a link to something more neutral. :D

Reply:
👍